DataFormats/SoATemplate/SoA.md

```diff
@@ -0,0 +1,222 @@
+# Structure of array (SoA) generation
```

Comment:
How about renaming the file as `README.md`?

Reply:
:+1: